Tutorials/index.md

```diff
@@ -68,6 +68,10 @@ Learn how to implement a basic register/login functionality on your website, hid
 
 This tutorial explains how to host multiple sites from one project/installation of Umbraco.
 
+## [Cache Static Assets](Cache-Static-Assets/)
+
+This tutorial explains how to cache static assets in Umbraco.
```

Comment:
I don't think the article qualifies as a tutorial 🤔 Or, if that's the intention, I think some more explanation and detail should be added to the article.

In any case, perhaps this should reflect the content in the article a little better:

```suggestion
This sample shows how to cache static assets in Umbraco.
```